### PR TITLE
Feat/secure field

### PR DESCRIPTION
**Pull request by kari0003**

### CODE REVIEW

File: Sources/PaylikeSDK/Views/Components/CardValidationCodeField.swift [hunk 1]
```diff
@@ -10,16 +10,15 @@ import SwiftUI
 struct CardValidationCodeField: View {
     @Binding public var cvc: String
     public var isValid: Bool
-    @State public var isEditing: Bool = false
```
Comment (kocsislaci) [inline Sources/PaylikeSDK/Views/Components/CardValidationCodeField.swift:13]: Why is this deleted? I mean this whole isEditing check

**Review by kocsislaci — APPROVED** [review]

File: Sources/PaylikeSDK/Views/Components/CardValidationCodeField.swift (hunk 1 above)
Comment (kari0003) [reply Sources/PaylikeSDK/Views/Components/CardValidationCodeField.swift:13]: onEditingChanged is not a SecureField property, so it cannot work the same way as textField
An alternative might be to use onSubmit or onCommit: https://developer.apple.com/documentation/swiftui/view/onsubmit(of:_:)
